### PR TITLE
Profilepage

### PR DESCRIPTION
**Pull request by CastorK**
Added a simple profile page that lists your ongoing bookings and upcoming bookings. The page also has a statistics section that at the moment only lists total time booked but can easily be expanded with more stats about the user, e.g. money spent on bookings

### CODE REVIEW

File: fars/booking/templates/profile.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    <div class="row text-center">
+      <h3 class="col-12">{% trans "Your Bookings" %}</h3>
+    </div>
+    <!-- Ongoing bookings -->
+    <div class="row">
+      <h5 class="col-12">{% trans "Ongoing bookings" %}<hr></h5>
+      {% for b in ongoing_bookings %}
+      <div class="col-10 offset-1 profile-event">
+        <div class="row">
+          <div class="col-6 offset-1 font-weight-bold">
```
Comment (tlangens) [inline fars/booking/templates/profile.html]: This is probably supposed to be col-3

File: fars/booking/templates/profile.html [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+            </div>
+        </div>
+      </div>
+      {% endfor %}
+    </div>
+    <!-- Future bookings -->
+    <div class="row">
+      <h5 class="col-12">{% trans "Upcoming bookings" %}<hr></h5>
+      {% for b in future_bookings %}
+      <div class="col-10 offset-1 profile-event text-center">
```
Comment (tlangens) [inline fars/booking/templates/profile.html]: Why is the layout here not the same as the one above?

File: fars/booking/templates/profile.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+          <div class="col-6">
+              {{ b.bookable }}
+          </div>
+        </div>
+        <div class="row">
+            <div class="col-3 offset-1 font-weight-bold">
+                {% trans "Start:" %}
+            </div>
+            <div class="col-8">
+                {{ b.start | date:"Y-m-d" }} {{ b.start | time:"H:i"}}
```
Comment (tlangens) [inline fars/booking/templates/profile.html]: Why not use the default rendering here as everywhere else?

File: fars/booking/templates/profile.html (hunk 2 above)
Comment (CastorK) [reply fars/booking/templates/profile.html]: Forgot to update the ongoing block aswell

File: fars/booking/templates/profile.html (hunk 1 above)
Comment (CastorK) [reply fars/booking/templates/profile.html]: Don't understand which line you mean

Comment (tlangens) [reply fars/booking/templates/profile.html]: Commented line, i.e. 20. I believe it should be same as line 28.

Comment (CastorK) [reply fars/booking/templates/profile.html]: Not relevant anymore then. Ongoing bookings now use the same design as future bookings

Comment (tlangens) [reply fars/booking/templates/profile.html]: I think I liked the previous design better :/
Not too fond of text-center

**Pull request merged**